README.md

```diff
@@ -119,6 +119,33 @@ pip install -e .
 
 ### Running dot
 
+#### Graphical interface
+
+To use the graphical interface of dot run
+
+```bash
+conda activate dot

```

Comment:
can we remove the need for ALL terminal instructions for the GUI? it defeats the purpose of building a GUI for non-technical users

Reply:
what I'd like to see is a guide for people to install the GUI just by download and clicks

Reply:
@Ghassen-Chaabouni I was asking about the need for conda. is that a requirement for installation?

Reply:
We won't need it to run the executable

Reply:
If you are asking for the case without the executable, then we need it to run the gui

Reply:
all clear, LGTM